docs/blog_post.md

```diff
@@ -0,0 +1,56 @@
+
+**The data and code used for this project are [publicly available](https://github.com/uptake/autofocus) so that anyone interested in developing the relevant skills can get involved.**
+
+## Impact
+
+> It was like they gifted us 10 interns who could work around the clock. Autofocus enables us to get on top of our backlog of images and deliver valuable insights on local wildlife now.
+>
+> --Urban Wildlife Institute researcher Mason Fidino, PhD <!--TK image of Mason-->
+
+Approximately 10-15 percent of the images that the Urban Wildlife Institute's cameras capture contain humans rather than the wildlife that is relevant to the institute's research. Those images are a major nuisance because the institute wishes to respect privacy by filtering out those images before posting the rest online for volunteers to label.
+
+![](human_example.JPG)
+<!--TK image caption formatting-->
+The Urban Wildlife Institute's cameras often capture images of people jogging or walking dogs, or (as in this case) images of the researchers themselves stopping by to service them.
+
+To address this problem, Uptake's philanthropic arm Uptake.org developed the first version of a program called Autofocus that can classify an image as containing a human or not. That program proved to be sufficiently accurate to remove approximately 50% of human-containing images with a negligible rate of false positives. **On one batch of images from Spring 2016, the Urban Wildlife Institude was able to use that model to filter out roughly 3,700 images.** "It was like they gifted us 10 interns who could work around the clock," said Urban Wildlife Institute researcher Mason Fidino. "Autofocus enables us to get on top of our backlog of images and deliver valuable insights on local wildlife now."
+
+**Data scientists at ShopRunner contributed to a second version of Autofocus that not only appears to increase human label accuracy, but also provides labels for eighteen additional kinds of animals.** The accuracy of that second version is currently under review, but preliminary results suggest that it is sufficient to automate classification of a large percentage of incoming images at least for common animal types such as deer and raccoons, further improving the speed of the Urban Wildlife Institute's research.
+
+## Methods
+
+![](https://brooksbrothers.scene7.com/is/image/BrooksBrothers/SX00251_HEATHER-GREY?$bbproductimages$)
+<!--TK get a fashion image that we definitely have permission to use and adjust caption accordingly-->
+<!--TK image caption formatting-->
+ShopRunner data scientists repurposed for Autofocus work that they did to develop a system that can classify this image as showing a gray, melange, knee-length, sleeveless dress for ShopRunner's iOS shopping app District.
```

Comment:
This sentence is a little confusing. Perhaps the 'for' before Autofocus is not supposed to be there?

Right now I'm not sure if the wildlife classifier got reporposed from the clothes classifier or vice versa

Reply:
I think it should be something like "For Autofocus, ShopRunner data scientists repurposed work that they did to develop a system that can classify this image as showing a gray, melange, knee-length, sleeveless dress for ShopRunner's iOS shopping app District."